src/org/jitsi/impl/neomedia/MediaUtils.java

```diff
@@ -53,7 +54,7 @@
      * defined in RFC 3551.
      */
     private static final Map<String, String> jmfEncodingToEncodings
-        = new HashMap<String, String>();
+        = new ConcurrentHashMap<>();
```

Comment:
That seems unnecessary because jmfEncodingToEncodings is modified only during the static constructor of MediaUtils.